assets/tests/components/mapPage/routePropertiesCard.test.tsx

```diff
@@ -0,0 +1,356 @@
+import React from "react"
+import { render, screen } from "@testing-library/react"
+import "@testing-library/jest-dom"
+import RoutePropertiesCard, {
+  patternDisplayName,
+} from "../../../src/components/mapPage/routePropertiesCard"
+import { routePatternFactory } from "../../factories/routePattern"
+import routeFactory from "../../factories/route"
+import shapeFactory from "../../factories/shape"
+import stopFactory from "../../factories/stop"
+
+import { RoutesProvider } from "../../../src/contexts/routesContext"
+import userEvent from "@testing-library/user-event"
+
+const route66 = routeFactory.build({ id: "66" })
+
+describe("patternDisplayName", () => {
+  test("When not formatted correctly with ' - '", () => {
+    const routePattern = routePatternFactory.build({
+      name: "BAdFormatted RP",
+    })
+
+    expect(patternDisplayName(routePattern)).toEqual({
+      name: routePattern.name,
+      description: "",
+    })
+  })
```

Comment:
**suggestion:** I guess this is more a comment on the application code than the test but this got me thinking - I think ideally we would want to at least display something in this case, even if it's not exactly what we want. I worry that displaying nothing when the route pattern name isn't formatted to match our expectations introduces too much of a dependency on the details of the GTFS data.

Reply:
Ahh, okay, makes sense. It's clear, I just didn't read carefully enough. 🙂